texar/tf/data/tokenizers/xlnet_tokenizer_test.py

```diff
@@ -19,9 +19,9 @@ class XLNetTokenizerTest(tf.test.TestCase):
     def setUp(self):
         self.tmp_dir = tempfile.TemporaryDirectory()
         self.SAMPLE_VOCAB = maybe_download(
-            'https://github.com/gpengzhi/pytorch-transformers/blob/master/'
-            'pytorch_transformers/tests/fixtures/test_sentencepiece.model'
-            '?raw=true', self.tmp_dir.name)
+            'https://github.com/huggingface/transformers/blob/master/'
```

Comment:
Add a comment to acknowledge downloading from huggingface for test

`# Use the test sentencepiece model downloaded from huggingface transformers`